### ISSUE TITLE
locahost not works

### ISSUE THREAD

**Issue by Juanmabs22** [FabianLauer/chrome-vs-code#5]
Hello Fabian!

I just looking for a extension like this and i appear in my life, so THANKS!

Currently i am developing my Final Degree Project on Vue.Js but not works:
I'm not sure how to debug o fix that.

![image](https://user-images.githubusercontent.com/5569316/37840429-51b7dcda-2ebd-11e8-8401-db4fbc9817e6.png)

THANKS!!

**Comment by FabianLauer** [comment]
Could you please have a look in VS Code's output panel, specifically in the "VS Code Browser" log and post its content when you try to open your page?

Plus, please post the full URL you're trying to open. Not sure how stable the URL resolution is.

I just did a quick test with a node.js server on `http://localhost:3000`. For me, the browser displays a "Server Error" and the output panel contains this:

```
(server) [proxy: error] http://localhost:3000 : Error: unable to verify the first certificate
```

It's obviously another error than you have, but it might still be related.

Thanks for the report! :slightly_smiling_face:

**Comment by Juanmabs22** [comment]
The full URL is http://localhost:3000/#/config/Microcontrolador but is s SPA so http://localhost:3000/ is the same.

```
(server) [proxy: 200] https://code.visualstudio.com/assets/images/apple-logo.svg 
(server) [proxy: error] http://localhost:3000 : Error: connect ECONNREFUSED 127.0.0.1:80 
(server) [proxy: error] http://localhost:3000 : Error: connect ECONNREFUSED 127.0.0.1:80 
(server) [proxy: error] http://localhost:3000 : Error: connect ECONNREFUSED 127.0.0.1:80 
(server) [proxy: error] http://localhost:3000 : Error: connect ECONNREFUSED 127.0.0.1:80 
```
Thanks! 👍

**Comment by Juanmabs22** [comment]
The plugin always try to use the port 80, changed to 80 on my project and now works better:
But now the page not loads, i tried with the Vue demo project, but stills not working:

My Vue project:
```
(server) [proxy: 200] http://localhost 
(server) [proxy: 200] http://localhost/static/css/font-awesome.min.css 
(server) [proxy: 200] http://localhost/static/css/simple-line-icons.css 
(server) [proxy: 200] http://localhost/runtime~main.ba0642ed3d7868e2eec5.js 
(server) [proxy: 200] http://localhost/commons.bundle.js 
(server) [proxy: 200] http://localhost/static/css/style.css 
(server) [proxy: 200] http://localhost/main.bundle.js 
(server) [proxy: 200] http://localhost/runtime~vendor.ba0642ed3d7868e2eec5.js 
(server) [proxy: 200] http://localhost/vendor.bundle.js 
(server) [proxy: 200] http://localhost/static/img/favicon.png 
(server) [404 -> proxy]: http://localhost/__webpack_hmr 
(server) [proxy: 200] http://localhost/__webpack_hmr 
```

Hello World Vue:
```
(server) [proxy: 200] http://localhost 
(server) [proxy: 200] http://localhost/app.js 
```

**Comment by khanenyao** [comment]
@FabianLauer i have the same problem. Do you have solution for that?

**Comment by pavittarx** [comment]
have you tried using 127.0.0.1 instead of localhost? most of the time, using 127.0.0.1 resolves quite a number of problems.

**Comment by khanenyao** [comment]
@Juanmabs22 Yes I have, but still not working for me...  I use custom port 3000, is it OK?

**Comment by bastienlemaitre** [comment]
Same issue...

**Comment by Alber70g** [comment]
It should be working on a non-80 port. It shouldn't default to port 80 when there's a port present

**Comment by FabianLauer** [comment]
Should be fixed by #10. There's more issues related to the URL bar, but they're outside the scope of this bug.

Thanks for your input guys!
